src/git.ts

```diff
@@ -842,26 +899,19 @@ export function handleGitLog(_actionData: ActionData): string | undefined {
 
 export function handleGitBlame(actionData: ActionData): string | undefined {
     assert(repo);
-
     const filePath: string = actionData.params.filePath;
-
-    // Normalize the file path if provided; otherwise, use wildcard.
-    const stageFiles: string = filePath ? getNormalizedRepoPathForGit(filePath) : '*';
-
-    // Compute an absolute path. If the stageFiles is already absolute, use it.
-    // Otherwise, join it with the repository's root path.
-    let absolutePath: string;
-    if (path.isAbsolute(stageFiles)) {
-        absolutePath = stageFiles;
-    } else {
-        absolutePath = path.join(repo.rootUri.fsPath, stageFiles);
+    if (!isPathNeuroSafe(filePath)) {
+        NEURO.client?.sendContext('The provided file path is not allowed.');
+        return;
     }
+    // Normalize the file path if provided; otherwise, use wildcard.
+    const absolutePath: string = getAbsoluteFilePath(filePath);
```

Comment:
`isPathNeuroSafe` only works with absolute paths (this isn't documented because I forgot)

```suggestion
    // Normalize the file path if provided; otherwise, use wildcard.
    const absolutePath: string = getAbsoluteFilePath(filePath);
    if (!isPathNeuroSafe(absolutePath)) {
        NEURO.client?.sendContext('The provided file path is not allowed.');
        return;
    }
```